CONTRIBUTING.md

```diff
@@ -0,0 +1,33 @@
+**Thanks for taking the time to contribute! ❤️**
+
+All types of contributions are encouraged and valued, no matter if it's a bug
+report 🐛, a feature request 💡, or a Pull Request 🚀.
+
+- **❓ I have a question:** [Open a Q&A Discussion]
+- **🐛 I found a bug:** [Open an Issue]
+- **💡 I have an idea:** [Open an Idea Discussion]
+- **💻 I want to code:** See below
+
+⚠️ You must never report security 🔒 related issues, vulnerabilities or bugs
+including sensitive information to the issue tracker, or elsewhere in public.
+Instead sensitive bugs must be sent by email to spenserblack01@gmail.com.
+
+## Contributing code
+
+1. 🔀 Fork the repo
+2. 💻 Open the repo in your editor
+3. 👨‍💻 Add your changes to your workspace
+4. 🔖 Commit & push your changes
+5. 🔁 Open a PR to get your changes merged
+6. 🧪 Let the tests run to make sure everything works
+7. 🚀 Profit!
+
+👩‍⚖️ When contributing code to this project, you must agree that you have authored
+100% of the content, that you have the necessary rights to the content and that
+the content you contribute may be provided under the project license.

```

Comment:
Is this legalese based on anything?
Going into lawyer mode:
1. What is "agree" in this statement? Is the act of contributing implicit agreement? 
2. So contributing on behalf of others (e.g. setting the commit author) is disallowed?

I think the license might be all the legal text we need. Any more could be confusing and scary to potential contributors.

Reply:
This contributing.md is heavily based on the https://generator.contributing.md/ text. In fact, this legal disclaimer is ripped right from it, just with a 🧑‍⚖️ emoji.

It might be overkill you're right